js/assessments/keywordStopWordsAssessment.js

```diff
@@ -52,6 +56,7 @@ module.exports = {
 	identifier: "keywordStopWords",
 	getResult: keywordHasStopWordsAssessment,
 	isApplicable: function( paper ) {
-		return paper.hasKeyword();
+		var isLanguageAvailable = getLanguageAvailability( paper.getLocale(), availableLanguages );
```

Comment:
When I set the site language in (for example) Italian and the user language in English, `paper.getLocale()` returns `en_US` while I guess the analysis should use the site language. I'm afraid this is because https://github.com/Yoast/wordpress-seo/pull/6215. We should probably pass both languages to the config and review what was done in https://github.com/Yoast/wordpress-seo/pull/6215